servers/pocket-router/renovate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
We dont need this, root level will handle the whole repo

Reply:
aha thank you